test/functional/mempool_accept.py

```diff
@@ -327,6 +328,52 @@ def run_test(self):
             rawtxs=[tx.serialize().hex()],
         )
 
+        # OP_RETURN followed by non-push
+        tx = tx_from_hex(raw_tx_reference)
+        tx.vout[0].scriptPubKey = CScript([OP_RETURN, OP_HASH160])
+        tx.vout = [tx.vout[0]] * 2
```

Comment:
what's this doubling doing?

Reply:
It creates two outputs to demonstrate that you can have more than one.

Reply:
right it already has coverage in https://github.com/bitcoin/bitcoin/pull/32359/files/47f9565c15faec0ce35a17c68ade85af1dc41677..8fd09d622d161eb554f080a52735832119cb35e5#diff-a99d72c0ed66c256169e92327e04ab223e71f5ef598e14aac0ff44dc2a1194daR351